ms365_accountcreator/templates/index.html

```diff
@@ -118,11 +121,9 @@ <h1>{{ gettext('Account Registration for Microsoft Teams for University of Stutt
 
         function consentSubmit(event) {
             event.preventDefault();
-            const mailConsent = document.getElementById('input-consent-mail').checked;
-            const teamsConsent = document.getElementById('input-consent-teams').checked;
-            if(!(mailConsent && teamsConsent)) {
-                alert("{{ gettext('Consent to both parts needed') }}")
-                return false;
+
+            if (!document.getElementById('email-form').reportValidity()) {
```

Comment:
I would prefer (also) keeping the `checked` check, as that does not rely on the validity criteria being correct